Bottom: df4460d78f61e2b742a336d7f0bd69e2473f4510
Top:    736fc0a082efd215f025e71472f8e570c525fcf2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-24 10:25:56 +0000

Refresh of add-netbyname-to-the-select-ac

---

diff --git a/src/select.c b/src/select.c
index eb5598b..ebada31 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1020,18 +1020,19 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       SaveUndoSerialNumber ();
       MENU_LOOP (&PCB->NetlistLib);
       {
-	Cardinal i;
-	LibraryEntryType *entry;
-	ConnectionType conn;
-
-	/* Name[0] and Name[1] are special purpose, not the actual name*/
-	if (menu->Name && menu->Name[0] != '\0' && menu->Name[1] != '\0' &&
-	    REGEXEC (menu->Name + 2))
-	  {
-	    for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
-	      if (SeekPad (entry, &conn, false))
-		RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
-	  }
+        Cardinal i;
+        LibraryEntryType *entry;
+        ConnectionType conn;
+
+        /* Name[0] and Name[1] are special purpose, not the actual name*/
+        if (menu->Name && menu->Name[0] != '\0' && menu->Name[1] != '\0' &&
+            REGEXEC (menu->Name + 2))
+          {
+            for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
+              if (SeekPad (entry, &conn, false))
+                RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2,
+                             true, true);
+          }
       }
       END_LOOP;
       RestoreUndoSerialNumber ();
